crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -444,6 +444,8 @@ en:
       identify: 'Identify'
     form:
       allocate: 'Allocate'
+      confirm_allocate: 'Allocating the node will wipe all the disks attached to the node including the shared disks. Proceed?'
+      allocate_tooltip: 'Allocate this node according to the parameters in this form'
```

Comment:
It really feels to me we're adding a tooltip just to have a tooltip. Is there any useful information that we can provide in the tooltip? If not, do we really need a tooltip?